archinfo/arch.py

```diff
@@ -272,6 +279,12 @@ def translate_register_name(self, offset, size=None):
         except KeyError:
             return str(offset)
 
+    def get_register_offset(self, name):
+        try:
+            return self.registers[name][0]
+        except:
+            raise ValueError("Register %s does not exist!" % name)
+
```

Comment:
What was the purpose of adding this? It doesn't seem to add any value to just saying `arch.registers[name]`, and generally you do not want to separate the size from the offset.